Projects/hGraphQL/Sources/Models/ContractModels.swift

```diff
@@ -0,0 +1,273 @@
+import Foundation
+
+public struct ActiveContractBundle: Codable, Equatable {
+    public let contracts: [Contract]
+    public let id: String
+    public let movingFlowEmbarkId: String?
+
+    public init(
+        bundle: GraphQL.ActiveContractBundlesQuery.Data.ActiveContractBundle
+    ) {
+        contracts = bundle.contracts.map { .init(contract: $0) }
+        movingFlowEmbarkId = bundle.angelStories.addressChange
+        id = bundle.id
+    }
+}
+
+public struct hIcon: Codable, Equatable {
```

Comment:
hmm, not sure if I would instantly think of this as a model when I see it, maybe call it something else?